packages/web-components/src/progressbar/progressbar.styles.ts

```diff
@@ -0,0 +1,156 @@
+import { css } from '@microsoft/fast-element';
+import { display } from '@microsoft/fast-foundation';
+import {
+  borderRadiusMedium,
+  colorBrandBackground2,
+  colorCompoundBrandBackground,
+  colorNeutralBackground6,
+  colorPaletteDarkOrangeBackground3,
+  colorPaletteGreenBackground3,
+  colorPaletteRedBackground3,
+} from '../theme/design-tokens.js';
+
+/** Text styles
+ * @public
+ */
+export const styles = css`
+  ${display('flex')}
+
+  :host {
+    --progress-bar-thickness: 2px;
+    --progress-bar-radius: ${borderRadiusMedium};
+    --progress-bar-color: ${colorCompoundBrandBackground};
+    --progress-speed: 3s;
```

Comment:
@miroslavstastny how does Fluent approach enabling customization like this?

Reply:
Based on @miroslavstastny's guidance, I'm happy removing and fixing this. If we do keep these in for components, we should try to figure out a way to add these to the component documentation somewhere.

But... is it weird that the web component controls would have these explicit overrides, but the React ones won't?

Reply:
FUIR9 currently does not support style overrides using "component variables" - there is currently a draft RFC discussing the possible options: https://github.com/microsoft/fluentui/pull/25333.

I am ok with this approach for now, but I would not advertise this to partners until we agree on the approach.

_Perhaps he will need to reopen this question once we will have a parent-child component where the parent needs to style its child. For now we agreed that this is user's responsibility to set appropriate attributes on the children explicitly, but we might eventually hit a case where it will not be possible._

Reply:
Sounds good. If we're not going to advertise it to partners, let's remove it until the RFC is settled and we can implement a standard and consistent approach. Part of me fears if we leave this in here now, it'll be too easy to forget about in the future when we do it the correct way.

Reply:
Removed, but it ended up increasing the size of the CSS file.